js/app.js

```diff
@@ -0,0 +1,61 @@
+let zombieTypes = {
+    dead: [],
+    small: [],
+    mad: [],
+    strong: [],
+};
+zombiesData.forEach(function(item) {
+    switch(typeof item) {
+        case 'number':
+            if (item > 0 && item < 11) {
+                zombieTypes.small.push(item);
+            } else if (item > 10 && item < 21) {
+                zombieTypes.mad.push(item);
+            } else if (item > 20) {
+                zombieTypes.strong.push(item);
+            } else {
+                zombieTypes.dead.push(item);
+            }
+            break;
+        default:
+            zombieTypes.dead.push(item);
+            break;
+    }
+});
+
+formatResult(zombiesData, '.total-zombies');
+insertResult(zombieTypes, 'dead', '.dead-zombies');
+insertResult(zombieTypes, 'small', '.small-zombies');
+insertResult(zombieTypes, 'mad', '.mad-zombies');
+insertResult(zombieTypes, 'strong', '.strong-zombies');
+
+document.addEventListener("DOMContentLoaded", function(){
```

Comment:
"DOMContentLoaded" is a good solution in case if your script defined in <head> block of the html page
in this task you don't need to use this